src/test/kotlin/no/nav/syfo/oppfolgingstilfelle/bit/OppfolgingstilfelleBitSpek.kt

```diff
@@ -1412,18 +1381,64 @@ class OppfolgingstilfelleBitSpek : Spek({
 
             val oppfolgingstilfelleList = oppfolgingstilfelleBitList.generateOppfolgingstilfelleList()
             oppfolgingstilfelleList.size shouldBeEqualTo 2
+            val oppfolgingstilfelle = oppfolgingstilfelleList.first()
 
-            val firstTilfelleDuration = Period.between(
-                oppfolgingstilfelleList.first().start,
-                oppfolgingstilfelleList.first().end,
-            ).days
+            val firstTilfelleDuration = oppfolgingstilfelle.start.until(oppfolgingstilfelle.end, ChronoUnit.DAYS)
             firstTilfelleDuration shouldBeEqualTo 0
+            oppfolgingstilfelle.antallSykedager shouldBeEqualTo 1
 
-            val secondTilfelleDuration = Period.between(
-                oppfolgingstilfelleList.last().start,
-                oppfolgingstilfelleList.last().end,
-            ).days
+            val oppfolgingstilfelleLast = oppfolgingstilfelleList.last()
+            val secondTilfelleDuration = oppfolgingstilfelle.start.until(oppfolgingstilfelle.end, ChronoUnit.DAYS)
             secondTilfelleDuration shouldBeEqualTo 0
+            oppfolgingstilfelleLast.antallSykedager shouldBeEqualTo 1
+        }
+        it("should return 1 Oppfolgingstilfelle with correct Sykedager even if multiple ferieperioder") {
```

Comment:
Mens jeg husker det: Går det an å teste hva som skjer hvis man har én arbeidsdag midt i en ferie og så går over 16 dager?
Altså `syk1 -> syk2 -> ferie -> arbeid -> ferie*16`
Det ser ut til at før ville vi fått et tilfelle som varte fra syk1 til syk2, men nå vil vi også få med den første biten med ferie? 🤔

Reply:
Ja, det kan være det blir en forskjell på tom'en i et slikt tilfelle. Men sånn må det vel nesten bli for feriedager som teller som sykedager?

Reply:
Jeg tror det blir riktig sånn du har gjort det nå! Men greit å få med en test på det 👍